### PR TITLE
[MGX-393] PolkadotMVR - xcm tests & xcm_config fine tuning

### PR DESCRIPTION
**Pull request by mateuszaaa**

### CODE REVIEW

File: runtime/integration-test/src/xcm/kusama_xcm_transfer.rs
Comment (vanderian) [inline runtime/integration-test/src/xcm/kusama_xcm_transfer.rs:1]: why are these tests commented out ?

File: mvr/runtime/mangata-polkadot/src/lib.rs [hunk 1]
```diff
@@ -365,9 +381,14 @@ impl parachain_info::Config for Runtime {}
...
 
+parameter_types! {
+	pub const FixedReserveAssetTransferTrapCost : Balance = 10_000_000_000; // 0.01 DOT
+}
+
 impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
-	type XcmExecutor = XcmExecutor<XcmConfig>;
+	type XcmExecutor =
+		xcm_config::ExecutorWrapper<XcmExecutor<XcmConfig>, FixedReserveAssetTransferTrapCost>;
```
Comment (vanderian) [inline mvr/runtime/mangata-polkadot/src/lib.rs:391]: sooo, the wrapper has to be used only here right,  the other pallets in xcm_config can use base XcmExecutor ?

File: mvr/runtime/mangata-polkadot/src/lib.rs [hunk 2]
```diff
@@ -148,6 +149,22 @@ impl WeightToFeePolynomial for WeightToFee {
...
+pub fn mgx_per_second() -> u128 {
+	let base_weight = Balance::from(ExtrinsicBaseWeight::get().ref_time());
+	let base_per_second = (WEIGHT_REF_TIME_PER_SECOND / base_weight as u64) as u128;
+	base_per_second * base_tx_in_mgx()
+}
+
+pub fn dot_per_second() -> u128 {
+	mgx_per_second() / DOT_MGX_SCALE_FACTOR_UNADJUSTED as u128
+}
+
```
Comment (vanderian) [inline mvr/runtime/mangata-polkadot/src/lib.rs:167]: what is the token setup in the MVR ?
similar to what we have now ? 0 native, 4 relay
which pays onchain fees ?

Comment (mateuszaaa) [reply mvr/runtime/mangata-polkadot/src/lib.rs:167]: yep, i kept the same ids as on rococo/kusama

File: mvr/runtime/mangata-polkadot/src/lib.rs (hunk 1 above)
Comment (mateuszaaa) [reply mvr/runtime/mangata-polkadot/src/lib.rs:391]: exactly

File: runtime/integration-test/src/xcm/kusama_xcm_transfer.rs
Comment (mateuszaaa) [reply runtime/integration-test/src/xcm/kusama_xcm_transfer.rs:1]: fixed

**Review by vanderian — DISMISSED** [review]

**Pull request merged**